server/@types/hmppsAuth/index.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface HmppsAuthUser {
+  name: string
+  activeCaseLoadId: string
+}
```

Comment:
Probably needs a few more details in this structure, but absolutely fine as  start.